game-core/src/main/java/games/strategy/engine/data/properties/AEditableProperty.java

```diff
@@ -10,12 +10,13 @@
  */
 public abstract class AEditableProperty implements IEditableProperty, Serializable, Comparable<AEditableProperty> {
```

Comment:
Is there anything speaking against Renaming this into `EditableProperty` or `AbstractEditableProperty` if that's taken already

Reply:
f76a3fbbd  I went with `AbstractEditableProperty` because we'll probably want to rename `IEditableProperty` to `EditableProperty` in the future.